src/main/java/com/recruitment/project/parking/parkingspaces/service/UserParkingVisitService.java

```diff
@@ -0,0 +1,29 @@
+package com.recruitment.project.parking.parkingspaces.service;
+
+import com.recruitment.project.parking.parkingspaces.domain.UserParkingVisit;
+
+import java.time.LocalDate;
+import java.util.List;
+
+public interface UserParkingVisitService {
+
+    final static long FIRST_HOUR_PARKING_PRICE_FOR_REGULAR = 1;
+    final static long SECOND_HOUR_PARKING_PRICE_FOR_REGULAR = 2;
+    final static double THIRD_AND_NEXT_HOURS_PARKING_PRICE_FOR_REGULAR_MULTIPLIER = 1.5;
```

Comment:
czy to jest dobre miejsce na te stałe?

Reply:
lepszym miejscem był by plik properties lub klasa Constans public final z prywantym konstruktorem.